docs/src/modules/sandbox/CreateReactApp.ts

```diff
@@ -0,0 +1,82 @@
+export const getHtml = ({ title, language }: { title: string; language: string }) => {
+  return `<!DOCTYPE html>
+<html lang="${language}">
+  <head>
+    <title>${title}</title>
+    <!-- Fonts to support Material Design -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <!-- Icons to support Material Design -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>`;
+};
+
+export const getRootIndex = (product?: 'joy-ui' | 'base') => {
+  if (product === 'joy-ui') {
+    return `import * as React from 'react';
+import ReactDOM from 'react-dom/client';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Demo from './demo';
+
+ReactDOM.createRoot(document.querySelector("#root")).render(
+  <CssVarsProvider>
```

Comment:
Should we add the wrapper to run in strict mode?

Reply:
Have we supported strict mode already? not sure but seems like it can be in another PR for improvements

Reply:
We run our docs in Strict Mode: https://github.com/mui/material-ui/blob/41a5f13678c5973d3088aa67550d8d8f2575e65c/docs/next.config.js#L241

Reply:
Nice, updated!